boards/arm/nrf9160_innblue21/doc/index.rst

```diff
@@ -0,0 +1,105 @@
+.. _nrf9160_innblue21:
+
+nRF9160-INNBLUE21
```

Comment:
Various comments I made in the PR for the other board (https://github.com/zephyrproject-rtos/zephyr/pull/23451#pullrequestreview-386044061) also apply to the documentation in this PR.

Can you please take a look there and let's resolve those issues, then apply any necessary changes here too?

Reply:
Hi @mbolivar-nordic / @ioannisg , I've made the required changes here and in the other board (https://github.com/zephyrproject-rtos/zephyr/pull/23451) and squashed commits as well.. 

Can you recheck and let me know if there is anything else we need to add to the board defs ? Thanks..

Reply:
Thanks!

Can you please clean up the commit log? There's no need to include the changes made in response to each round of review in the commit log. Just describe the changes made by the commit.

I know the git squash UI opens your editor with the commit messages concatenated, but the idea is that you then edit them together into a single commit message.

Reply:
Done.. There was also another problem in the defconfig where we were modifying an non-selectable symbol (NRFX_UART0) which was causing shippable to complain.. I've fixed that and hopefully we should be good to go here..